astroquery/sdss/field_names.py

```diff
@@ -66,21 +66,21 @@ def _columns_json_to_table(jsonobj):
 # below here are builtin data files
 def _load_builtin_table_fields():
     key1 = ('PhotoObjAll',
-            'http://skyserver.sdss.org/dr12/en/tools/search/x_sql.aspx')
+            conf.skyserver_baseurl + '/dr12/en/tools/search/x_sql.aspx')
```

Comment:
I wonder whether it makes any sense to remove hardwiring DR12 from here. Are the field names same for all the DRs?

Reply:
The database column names should be pretty much the same from DR8 onward.  However, the URL paths changed in DR12, mostly `.asp` to `.aspx`.